MultiLangProgGuide/MultiLangProgGuide.rst

```diff
@@ -327,6 +327,17 @@ Chapter 8 (Intrinsic Special Variables) of `GT.M Programmers Guide
 useful to M application code, others are more generally useful and
 documented here.
 
+-------
+$tlevel
+-------
+
+Application code can read the intrinsic special variable ``$tlevel``
+to determine whether it is executing inside a
+transaction. ``$tlevel>0`` means that it is inside a transaction, and
+``$tlevel>1`` means that it is inside a nested transaction. Note that
+a transaction can be started explicitly, e.g., by calling
+`ydb_tp_s()`_ ,or implicitly by a trigger on a database update.
```

Comment:
Should we mention "a ydb_set_s() or ydb_kill_s() etc." instead of "a database update"?

Reply:
Done